WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -23,35 +22,100 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Advanced.WalletCoins;
 [NavigationMetaData(Title = "Wallet Coins (UTXOs)")]
 public partial class WalletCoinsViewModel : RoutableViewModel
 {
-	private readonly WalletViewModel _walletViewModel;
 	private readonly IObservable<Unit> _balanceChanged;
-	private readonly ObservableCollectionExtended<WalletCoinViewModel> _coins;
-	private readonly SourceList<WalletCoinViewModel> _coinsSourceList = new();
-	[AutoNotify] private FlatTreeDataGridSource<WalletCoinViewModel>? _source;
+	private readonly WalletViewModel _walletViewModel;
+	private ObservableAsPropertyHelper<bool>? _isAnySelected;
```

Comment:
And you don't have to deal with nullable.

```suggestion
	private ObservableAsPropertyHelper<bool> _isAnySelected = ObservableAsPropertyHelper<bool>.Default();
```

Reply:
Lovely! almost forgot there's a static member for default value :)
Thanks!